Tests/Components/TextTests.cs

```diff
@@ -57,5 +57,15 @@ public void Should_render_raw_text_with_whitespace()
 
             AssertHelpers.HtmlFileAssert("Components.Outputs.TextRawWhitespace.html", result);
         }
+
+        [Fact]
+        public void Should_render_text_with_html_and_whitespace()
+        {
+            var source = @"<mj-text>This <strong>should</strong> respect <strong>whitespaces.</strong> after the <strong>HTML Tag</strong></mj-text>";
+
+            var result = TestHelper.Render(source);
+
+            AssertHelpers.HtmlFileAssert("Components.Outputs.TextWithHtmlAndWhitespace.html", result);
```

Comment:
Have you tested how strict the HtmlFileAssert is about whitespaces? Perhaps we should just make a normal "Contains" here.

Reply:
Good point, I'll use contains.